packages/manager/modules/server-sidebar/src/sidebar.constants.js

```diff
@@ -369,6 +369,33 @@ export const DEDICATED_NETWORK_CONFIG = {
   ],
 };
 
+export const STORAGE_BACKUP_CONFIG = {
+  id: 'storage_backup',
+  forceDisplaySearch: true,
+  app: [DEDICATED],
+  feature: 'netapp',
+  icon: 'ovh-font ovh-font-cloudnas',
```

Comment:
Waiting confirmation for icons (here same icon is used three time in a row).

Reply:
PR to add icon: https://github.com/ovh/ovh-ui-kit/pull/725